src/testRunner/unittests/tsbuild/sample.ts

```diff
@@ -358,18 +358,33 @@ export class cNew {}`);
             });
         });
 
+        const coreChanges: TscIncremental[] = [
+            {
+                buildKind: BuildKind.IncrementalDtsChange,

```

Comment:
This is the change that's made on top of sample (from tests/projects/sample1) and  the baseline goes into` incremental-declaration-changes/scenario name`. The baseline contains output on the command line and the changes in file system